src/themes/DiamondTheme.ts

```diff
@@ -1,47 +1,86 @@
-import { createTheme, ThemeOptions } from '@mui/material/styles';
+import { createTheme, Theme } from "@mui/material/styles";
 
-import {BaseThemeOptions} from "./BaseTheme";
+import { BaseThemeOptions } from "./BaseTheme";
 
-const dlsLogoBlue = "#202740"
-const dlsLogoYellow = "#facf07"
+const dlsLogoBlue = "#202740";
+const dlsLogoYellow = "#facf07";
 
-const DiamondTheme: ThemeOptions = createTheme({
+const DiamondTheme: Theme = createTheme({
   ...BaseThemeOptions,
   colorSchemes: {
     // https://zenoo.github.io/mui-theme-creator/
     light: {
       palette: {
         primary: {
           main: dlsLogoBlue,
-          // contrastText: '#ffffff',
-          // dark: '#161B2C',
-          // light: '#4C5266',
+          light: "#4C5266", // dark grey
+          dark: "#161B2C", // dark blue
+          contrastText: "#ffffff", // white
         },
         secondary: {
-            main: dlsLogoYellow,
-            // light: '#FBD838',
-            // dark: '#AF9004',
-            // contrastText: '#000000',
-        }
+          main: dlsLogoYellow,
+          light: "#FBD838", // light yellow
+          dark: "#AF9004", // dark yellow
+          contrastText: "#000000", // black
+        },
       },
     },
     dark: {
       palette: {
         primary: {
-          main: '#6175bd',  //lightened version of {dlsLogoBlue}
-          // light: '#8090CA',
-          // dark: '#435184',
-          // contrastText: '#ffffff',
+          main: "#6175bd", //lightened version of {dlsLogoBlue}
+          light: "#8090CA", // lighter blue
+          dark: "#435184", // mid blue
+          contrastText: "#ffffff", // white
         },
         secondary: {
           main: dlsLogoYellow,
-          // light: '#FBD838',
-          // dark: '#AF9004',
-          // contrastText: '#000000',
-        }
+          light: "#FBD838", // light yellow
+          dark: "#AF9004", // dark yellow
+          contrastText: "#000000", // black
+        },
+      },
+    },
+  },
+  components: {
+    MuiButton: {
+      styleOverrides: {
+        root: ({ theme }) => ({
+          borderBottom: "2px solid rgba(0,0,0,0.2)",
```

Comment:
I don't think this is correct, as it's setting the bottom line of all the different variants of the button to have something that looks like a shadow. e.g. the contained one, ins't contained anymore.

Reply:
I don't think we really want this border anyway. Could remove this line and then add `"&.MuiButton-contained": {},` below:

`root: ({ theme }) => ({
          textTransform: "none",
          "&.MuiButton-contained": {},
          "&.default": {`
          
Something like this would prevent the "contained" variant being overridden and can then be modified as we want. It still applies the text transform (text not changed to uppercase) to everything.